gyp_xwalk

```diff
@@ -220,7 +220,10 @@ if __name__ == '__main__':
   args = list(set(args))
   delist = []
   ip_media_codecs = False # Default: no third-party codecs be build in.
+  enable_web_audio = True
   for arg in args:
+    if arg == '-Ddisable_web_audio=1':
+      enable_web_audio = False
```

Comment:
We can remove this enable_web_audio, just add args.append('-Ddisable_web_audio=1') below the "if gyp_vars_dict.get('OS') == 'android'" condition.